core/server/api/v2/utils/serializers/output/utils/mapper.js

```diff
@@ -11,6 +11,9 @@ const mapPost = (model, frame) => {
     if (utils.isContentAPI(frame)) {
         date.forPost(jsonModel);
         members.forPost(jsonModel, frame);
+        if (!frame.original.query.include.includes('tags')) {
+            delete jsonModel.tags;
+        }
```

Comment:
Maybe better to move this condition inside the members module used here and behind the flag as above, so its all in once place. 😄